recipes-ni/sysconfig-settings/files/nisetembeddeduixml

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
Merge commits:
* initscripts-nilrt: remove nisetembeddeduixml
* sysconfig-settings: add nisetembeddeduixml

I would prefer a single commit which shows the `nisetembeddeduixml` script being moved.